### PR TITLE
Weather app clone

### PR DESCRIPTION
**Pull request by Arunlalpp**

### CODE REVIEW

File: src/components/CardTitle.js [hunk 1]
```diff
@@ -3,12 +3,11 @@ import React from "react";
 export default function CardTitle() {
   return (
     <div>
```
Comment (vonnue-sijin) [inline src/components/CardTitle.js:5]: What is the need for this `<div>`? If it is just a wrapping element using [React Fragments ](https://reactjs.org/docs/fragments.html).

File: src/components/CardTitle.js [hunk 2]
```diff
@@ -3,12 +3,11 @@ import React from "react";
...
-      <div className=" bg-slate-500">
-        <div className="w-[53%] ml-[12.5%] bg-white rounded-lg">
+      
           <div className="p-4 rounded-lg">
             <span className="text-black text-xl font-bold">Don't Miss</span>
-            <div className="flex flex-row gap-12">
-              <div className="">
+            <div className="flex flex-row gap-12 w-[95%]">
```
Comment (vonnue-sijin) [inline src/components/CardTitle.js:9]: Don't use this - 95 % there should be tailwind classes to achieve this. Connect with Nivya Peter if you have doubts she had done it before.

File: src/components/CardTitle.js [hunk 3]
```diff
@@ -3,12 +3,11 @@ import React from "react";
...
+              <div className="w-">
```
Comment (vonnue-sijin) [inline src/components/CardTitle.js:10]: What is this className "w-" do? It is an invalid class name, remove it.

Comment (vonnue-sijin) [inline src/components/CardTitle.js:10]: What is the need for this `<div>`? Extra divs will increase the time to paint the DOM (Nested divs)

File: src/components/CardTitle.js [hunk 4]
```diff
@@ -3,12 +3,11 @@ import React from "react";
...
                 <div>
                   <img className="rounded-lg" src="https://s.w-x.co/in-heat_14.jpg?crop=16:9&width=640&format=pjpg&auto=webp&quality=60" />
```
Comment (vonnue-sijin) [inline src/components/CardTitle.js:12]: Always provide `alt` attribute for image tags.

File: src/components/CardTitle.js [hunk 5]
```diff
@@ -27,7 +26,6 @@ export default function CardTitle() {
                 <div className="">
```
Comment (vonnue-sijin) [inline src/components/CardTitle.js:26]: Why empty classname?

File: src/components/Footer.js [hunk 6]
```diff
@@ -13,10 +9,92 @@ export default function Footer() {
...
+                    role="img"
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 320 512"
+                  >
+                    <path
+                      fill="black"
+                      d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
+                    ></path>
+                  </svg>
+                </a>
```
Comment (vonnue-sijin) [inline src/components/Footer.js:32]: Make this into a component and reuse below.

File: src/components/Navbar.js [hunk 7]
```diff
@@ -9,6 +9,7 @@ import Navlinks from "./Navlinks";
 import CardContainer from "./CardContainer";
 import IndianMap from "./IndianMap";
 import Footer from "./Footer";
+import CardTitle from "./CardTitle";
```
Comment (vonnue-sijin) [inline src/components/Navbar.js:12]: Please sort the imports. Also fetch farthest resource first.

File: src/components/CardTitle.js (hunk 4 above)
Comment (Arunlalpp) [reply src/components/CardTitle.js:12]: Thanks

File: src/components/CardTitle.js (hunk 2 above)
Comment (Arunlalpp) [reply src/components/CardTitle.js:9]: Oki, I just configured the tailwind theme file

**Pull request merged**